packages/core/src/external/fhir/export/string/resources/coverage.ts

```
@@ -0,0 +1,127 @@
+              cost.exception
+                .map(ex => {
+                  const exComponents = [
+                    formatCodeableConcept(ex.type, "Type"),
+                    formatPeriod(ex.period, "Period"),
+                  ].filter(Boolean);
+                  return exComponents.length > 0
+                    ? `Exception: ${exComponents.join(FIELD_SEPARATOR)}`
+                    : undefined;
+                })
+                .filter(Boolean)
+                .join(FIELD_SEPARATOR),
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)